corehq/apps/users/models.py

```diff
@@ -2256,6 +2247,30 @@ def get_device(self, device_id):
                 return device
 
 
+def update_fixture_status_for_users(user_ids, fixture_type):
+    from corehq.apps.fixtures.models import UserFixtureStatus
+    from dimagi.utils.chunked import chunked
+
+    now = datetime.utcnow()
+    for ids in chunked(user_ids, 50):
+        (UserFixtureStatus.objects
+         .filter(user_id__in=ids,
+                 fixture_type=fixture_type)
+         .update(last_modified=now))
+    for user_id in user_ids:
+        get_fixture_statuses.clear(user_id)
+
+
+@quickcache(['user_id'], lambda _: settings.UNIT_TESTING)
```

Comment:
@gbova just of curiosity, what does `lambda _: settings.UNIT_TESTING` do here?

Reply:
That's a `skip_arg` - it's called to determine whether to skip the cache.  That is, this means that the quickcache will not be used in testing.

Reply:
@gbova I think @mkangia's question here also illustrates a point—without a lot of familiarity (and confidence in your familiarity) with the argument structure of quickcache beyond the first mandator argument, it's pretty unclear what this second argument does. A way to make it a lot clearer (and what I recommend) is to (1) pass it as a keyword argument, and (2) use `user_id` instead of `_` to show (even though it's not used) that the argument to the lambda represents the `user_id`, reinforcing the (correct) notion that the arguments of the function you pass to `skip_arg` mirror those of the function you're decorating.

For this line that would mean replacing
```
@quickcache(['user_id'], lambda _: settings.UNIT_TESTING)
```
with
```
@quickcache(['user_id'], skip_arg=lambda user_id: settings.UNIT_TESTING)
```

(Also, I realize that you were just moving this code and/or following the style of the code you removed.)

Reply:
Thank you all..
also thank you @dannyroberts for adding the explanation.. 
I prefer keyword arguments over positional arguments.. for quick clarity by avoiding to lookup for the arguments being expected in the method being called.